### PR TITLE
Update eslint and node and universe

### PR DESCRIPTION
**Pull request by jonathanKingston**
I just wanted to update eslint so I can verify a future change which required a lot of changes.

- Travis needed updating to the latest node.
- Which broke npm installing, so I've updated the npm dependencies.
- Migrated grunt:execute to grunt:exec as the package doesn't work with the latest grunt.
- Reformatted the code automatically with eslint --fix
  - Two manual changes were added to add `return false` in the two some() filters in findRule (no idea why we have two matching files but hey)

### CODE REVIEW

File: src/trackers.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ class Trackers {
     }
 
     processTrackerList (data) {
-        for (let name in data) {
```
Comment (kdzwinel) [inline src/trackers.js:12]: all these years and it's still not obvious to me why const works here. I guess we are not changing value but destroying/creating new variable with every loop?

**Review by kdzwinel — COMMENTED** [review]
Code LGTM, but I'll have to come back to run it.

File: src/trackers.js (hunk 1 above)
Comment (jonathanKingston) [reply src/trackers.js:12]: Yeah exactly that.

We're stating that _name_ is non mutable but the for expression runs in isolation so we just redefine a new variable with called _name_ and give it a value.

Comment (jonathanKingston) [reply src/trackers.js:12]: I used to be kinda pro relaxing rules for ESLint here due to it being a little confusing syntactically. But unless we need mutability of the variable I'm happier now preventing accidental side effects.

File: Gruntfile.js [hunk 2]
```diff
@@ -59,10 +49,10 @@ module.exports = (grunt) => {
 
     grunt.registerTask('updateLists', 'Update data lists used by the grade calculation', [
         'exec:setup',
-        'execute:entityMap',
-        'execute:tosdr',
-        'execute:polisis',
-        'execute:trackersWithParentCompany'
+        'exec:entityMap',
+        'exec:tosdr',
```
Comment (kdzwinel) [inline Gruntfile.js:53]: this doesn't really work anymore - it tries to download https://tosdr.org/index/services.json which is 404

Comment (kdzwinel) [reply Gruntfile.js:53]: In fact, I'm pretty sure that the whole `updateLists` action can be removed together with the 'data-scripts' folder (that haven't been touched in 4y).

**Review by kdzwinel — APPROVED** [review]
LGTM

**Pull request merged**